etc/versions.cfg

```diff
@@ -0,0 +1,509 @@
+# THIS FILE IS GENERATED AUTOMATICALLY
```

Comment:
Gerou sem o `[versions]`, seria o caso de fazer `echo -e "[versions]\n# THIS FILE IS GENERATED AUTOMATICALLY"`?

Reply:
não gerou, sem o `[versions]` o `sort` que mexeu ai; vou tentar consertar.